modules/react/common/lib/utils/models.ts

```diff
@@ -1,6 +1,9 @@
 /* eslint-disable react-hooks/rules-of-hooks */
 import React from 'react';
 
+/**
+ * @deprecated
```

Comment:
should we add a message here as to what to use i guess not if they're using the new utilities